docker-compose.preview.yml

```diff
@@ -5,27 +5,30 @@ services:
             POSTGRES_USER: preview
             POSTGRES_PASSWORD: preview
             POSTGRES_DB: preview
-            MINIO_ROOT_USER: preview
-            MINIO_ROOT_PASSWORD: preview123
+            MINIO_ROOT_USER: minioadmin
+            MINIO_ROOT_PASSWORD: minioadmin
             ASSETS_UPLOAD_KEY: preview-different
             ASSETS_UPLOAD_SECRET_KEY: preview-different123
-            ASSETS_STORAGE_ENDPOINT: https://${PULLPREVIEW_PUBLIC_DNS}/assets
+            ASSETS_STORAGE_ENDPOINT: https://${PULLPREVIEW_PUBLIC_DNS}/a
             FLAGS: uploads:off,invites:off,disabled-actions:http+email
             ENV_NAME: sandbox
             DATACITE_API_URL: https://api.test.datacite.org
             DATACITE_REPOSITORY_ID: DATACITE_REPOSITORY_ID
             DATACITE_PASSWORD: DATACITE_PASSWORD
-    # minio-init:
-    #     restart: on-failure
-    #     environment:
-    #         MINIO_ROOT_USER: preview
-    #         MINIO_ROOT_PASSWORD: preview123
-    #         ASSETS_UPLOAD_KEY: preview-different
-    #         ASSETS_UPLOAD_SECRET_KEY: preview-different123
+
+    minio-init:
+        restart: on-failure
+        environment:
+            MINIO_ROOT_USER: minioadmin
+            MINIO_ROOT_PASSWORD: minioadmin
+            ASSETS_UPLOAD_KEY: preview-different
+            ASSETS_UPLOAD_SECRET_KEY: preview-different123
```

Comment:
i think i managed to fix minio-init causing the preview to fail by not having it as the last thing in the chain, but idk it does work now, but browser based uploads will not work, as you can't really put `minio` on a `subpath` eg `https://..../assets`. S3s signing algorithm doesnt like it